environments.ini

```diff
@@ -1539,6 +1539,46 @@ build_flags =
 custom_description = BLE gateway on the C3
 custom_hardware = AirM2M ESP32C3-CORE
 
+; Wemos Lolin C3 mini v2.1.0
+[env:lolin_c3_mini]
+;platform = ${com.esp32_c3_s3_platform} ; standard is @5.2.0, causes `assert rwble.c 261` error
+platform = espressif32@5.3.0  ; override to @5.3.0. Includes a fixed for the `assert rwble.c 261` error
+board = lolin_c3_mini
+board_build.partitions = min_spiffs.csv
+monitor_speed = 115200
+lib_deps =
+  ${com-esp.lib_deps}
+  ${libraries.wifimanager32}
+  ${libraries.ble}
+  ${libraries.decoder}
+build_flags =
+  ${com-esp.build_flags}
+  '-DZgatewayBT="BT"'
+; FastLED must be disabled. FastLED is not compatible with this board at the current moment.
+; The LED pins must still be defined, to avoid an error loop on pin definitions.
+; Available GPIOs are 0 - 8, 10, 20, 21.
+; See https://community.openmqttgateway.com/t/watchdog-kills-omg-on-esp32-c3-within-seconds/2192/15
+  '-DLED_SEND_RECEIVE=10'
+  '-DLED_INFO=6'
+  '-DLED_ERROR=7'
+  '-DTRIGGER_GPIO=9'
+  '-DNO_INT_TEMP_READING=true' ; No internal temperature on ESP32 C3 or S3
+  '-DGateway_Name="OpenMQTTGateway_ESP32C3_BLE"'
+; Serial output currently causes a delay on each message, if no serial listener is connected.
+; Therefore we disable serial output. Use the _with_serial environment if you
+; need serial output for debugging purposes.
+  '-DLOG_LEVEL=LOG_LEVEL_SILENT'  ; disable serial logging.
+  '-DWM_DEBUG_LEVEL=DEBUG_ERROR'  ; disable serial logging.
```

Comment:
You may try 0 instead of `DEBUG_ERROR`

Reply:
#  define WM_DEBUG_LEVEL 1 // valid values are: DEBUG_ERROR = 0

DEBUG_ERROR is the same as 0, no? But sure, i can change it to 0 instead. I will also update some comments so that the definition also matches the Seeed C3 board.